Install fails on latest OSMC 2022.09-1 on Pi3

I do use an NFS root for my OSMC installs so I am going to try a regular install to SD card today to see if I have better luck but I had errors about mupen64plus not installing and the kodi addon fails to launch.  It does seem like emulation station installed ok as I was able to exit kodi and launch that directly via the shell script.

Will update with error messages as fast as I can generate them.

This is where the mupen64 build bombs out on me.
```
../../src/Combiner.cpp: In member function ‘UpdateCombiner’:
../../src/DecodedMux.h:53:19: note: at offset 0 to object ‘aRGB0’ with size 1 declared here
   53 |             uint8 aRGB0;
      |                   ^
g++: fatal error: Killed signal terminated program lto1
compilation terminated.
lto-wrapper: fatal error: g++ returned 1 exit status
compilation terminated.
/usr/bin/ld: error: lto-wrapper failed
collect2: error: ld returned 1 exit status
make: *** [Makefile:469: mupen64plus-video-rice.so] Error 1
make: Leaving directory '/home/osmc/retrOSMCmk2/submodule/RetroPie-Setup/tmp/build/mupen64plus/mupen64plus-video-gles2rice/projects/unix'
current revision "2973f53"
last build revision 
/home/osmc/retrOSMCmk2/submodule/RetroPie-Setup/tmp/build/mupen64plus/GLideN64/projects/cmake /home/osmc/retrOSMCmk2/submodule/RetroPie-Setup/tmp/build/mupen64plus /home/osmc/retrOSMCmk2
```
and still it builds... slowly

--- a/resources/app-switcher.sh
+++ b/resources/app-switcher.sh
@@ -154,7 +154,10 @@ fs_setting=$(cat /home/osmc/.kodi/userdata/addon_data/script.launch.retropie/set
 
 # setup FIFO for communication
 if [[ ! -p $FIFO ]]; then
-  sudo -u osmc mkfifo $FIFO
+  # Recent kernels (4.19+) "disallow open of FIFOs ... not owned by the user in
+  #   world writable sticky directories"
+  # See https://kernelnewbies.org/Linux_4.19#Security
+  sudo mkfifo -m 660 $FIFO && sudo chown root:osmc $FIFO
 fi
 
 while true; do

